### PR TITLE
Adds secondary language processors for Vaurcae

### PR DESCRIPTION
**Pull request by feartheblackout**
Adds secondary language processors to Vaurcae, which allows them to select another language on top of their hive's language.

**NEEDS LORE/MAINTAINER APPROVAL BEFORE MERGING**

The current loadout point cost is 1, and the current included languages are Sinta'azaziba for the Mi'kuetz (potentially other Queenless factions) and Freespeak for the Zinos of the Underworld. If approved, these factors can all be adjusted as needed.

~apply for vaurca today~

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_xeno/vaurca.dm [hunk 1]
```diff
@@ -183,6 +183,22 @@
...
 
+/datum/gear/augment/auxiliary_processor
+	display_name = "secondary language processor"
+	description = "An augment that allows a vaurca to speak and understand a related language. These are only used by their respective groups, and act as an auxiliary to the hive's main language processor."
+	path = /obj/item/organ/internal/augment/language/mikuetz
+	cost = 1
+	sort_category = "Xenowear - Vaurca"
+	whitelisted = list(SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR, SPECIES_VAURCA_BREEDER, SPECIES_VAURCA_BULWARK)
+	flags = GEAR_NO_SELECTION
+
```
Comment (RustingWithYou) [inline code/modules/client/preference_setup/loadout/loadout_xeno/vaurca.dm:194]:
```suggestion
origin_restriction = list(/singleton/origin_item/origin/klatxatl, /singleton/origin_item/origin/mikuetz)
```

Comment (RustingWithYou) [reply code/modules/client/preference_setup/loadout/loadout_xeno/vaurca.dm:194]: Given the specific nature of these two, we may as well origin-lock it.

**Comment by feartheblackout** [comment]
oops

**Pull request closed without merging**